index.html

```diff
@@ -9,8 +9,8 @@ <h3 class="mk-masthead__content__sub-title">Bare metal host provisioning for Kub
 </section>
 <section class="mk-community-callout">
         <div class="mk-community-callout__images">
-          <img src="/assets/images/home-left-image.svg" alt="lightbulb and 3 persons">
-          <img src="/assets/images/home-right-image.svg" alt="laptop with metal³ logo">
+          <img src="/assets/images/home-left-image.svg" alt="">
```

Comment:
This was used to avoid issues when validating the website, is there any problem if it's left with the alternate text?